docs/reference/api-tutorials/robotoff.md

```diff
@@ -0,0 +1,6 @@
+About
+The Robotoff project is intended to complete missing information of products by prompting users to confirm predictions inferred by Artificial Intelligence algorithms. These algorithms are calculated based on "insights", which are facts about a product that have been extracted or deduced from the product pictures, ingredients, categories, labels, etc... 
+The OpenAPI documentation of the Robotoff API is located at : https://openfoodfacts.github.io/robotoff/references/api/#tag/Questions/operation/getQuestionsRandom 
+
+The more general Robotoff documentation is located at : 
+https://openfoodfacts.github.io/robotoff/
```

Comment:
I think this is more part of /intro/, don't you think so ?

Reply:
moved @alexgarel

Reply:
ok to merge ?